### PR TITLE
Adding IIS Administration link to TOC

### PR DESCRIPTION
**Pull request by terrimorton**
addresses [https://github.com/MicrosoftDocs/iis-docs/issues/9](https://github.com/MicrosoftDocs/iis-docs/issues/9)

### CODE REVIEW

File: iis/TOC.md [hunk 1]
```diff
@@ -821,3 +821,4 @@
 ## [Server Variables](web-dev-reference/server-variables.md)
 # [Contribute](contribute.md)
 # [Configuration Reference](configuration/index.md)
+# [IIS Administration](https://docs.microsoft.com/en-us/IIS-Administration/)
```
Comment (shirhatti) [inline iis/TOC.md]: Nit: I'd rather the unlocalized uri - https://docs.microsoft.com/IIS-Administration/

**Comment by shirhatti** [comment]
Looks like this also address #121

**Comment by shirhatti** [comment]
Looks like this table still doesn't render properly https://review.docs.microsoft.com/en-us/iis/install/installing-iis-7/iis-features-and-vista-editions?branch=terrimorton-patch-2

**Comment by terrimorton** [comment]
I've now removed the style attribute from the table element. This should hopefully fix the table rendering.

**Comment by bariscaglar** [comment]
The table is still broken due to colspan=x values. It should read colspan="x". I am sending a PR elsewhere to fix that.

**Pull request merged**